README.md

```diff
@@ -1 +1,1175 @@
-# eip-message-transformation
\ No newline at end of file
+[![Build Status](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)](https://travis-ci.org/sanethmaduranga/eip-message-transformation.svg?branch=master)
+
+# message-transformation---ballerina
+There are different ways of message transformation methods in EIP (Enterprise Integration Patterns). In this guide, we are focusing on 'content filter', 'claim check' and 'content enricher' message transformation methods between services using an example scenario.
+
+> This guide describes implementing three message transformation patterns using Ballerina programming language as simple steps.
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Deployment](#deployment)
+- [Observability](#observability)
+
+## What you’ll build
+When it comes to the data communication, the major challenge is formats of storage mechanisms vary among the systems. 
+
+![alt text](images/messagetrasformation.png)
+
+Also, the message producers and consumers use different techniques according to their requirement. So message transformation plays an important role in coupling those message producers and the message consumers. 
+Additionally,  the performance impact while message transformation is also an important fact in the real world. Here we discuss main three message transformation patterns in Integration as content filter, content enricher, and claim check.
```

Comment:
Remove extra spaces.

Reply:
Change to 'Additionally, the performance impact during message transformation is an important factor in the real world. Here, we discuss the three main message transformation patterns in integration, i.e., content filter, content enricher, and claim check."

Reply:
Done